ironfish-cli/src/commands/wallet/transactions/index.ts

```diff
@@ -72,15 +72,25 @@ export class TransactionsCommand extends IronfishCommand {
     const client = await this.connectRpc()
     await ui.checkWalletUnlocked(client)
 
+    const allAccounts = (await client.wallet.getAccounts()).content.accounts
+
     let accounts = flags.account
     if (flags['no-account']) {
-      const response = await client.wallet.getAccounts()
-      accounts = response.content.accounts
+      accounts = allAccounts
     } else if (!accounts) {
       const account = await useAccount(client, undefined)
       accounts = [account]
     }
 
```

Comment:
The column is populated with known account names. If the account name is not known it does not populate it.

The account name not being populated is considered "working" because these are externally not known accounts.